app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,8 @@
+class SessionsController < ApplicationController
+  def create
+    user = UsersFacade.login_user(request.env['omniauth.auth'])
+    session[:user_id] = user[:data][:id]
+
+    redirect_to '/dashboard'
+  end
+end
```

Comment:
we can create a private method:
def auth_hash
   request.env['omniauth.auth']
end

and call it on line 3

Reply:
Nice I can make that adjustment.